src/common/HTTPResponse.ts

```diff
@@ -67,38 +78,70 @@ export class HTTPResponse {
       : null;
   }
 
+  /**
+   * @internal
+   */
   _resolveBody(err: Error | null): void {
     return this._bodyLoadedPromiseFulfill(err);
   }
 
+  /**
+   * @returns The IP address and port number used to connect to the remote
+   * server.
+   */
   remoteAddress(): RemoteAddress {
     return this._remoteAddress;
   }
 
+  /**
+   * @returns The URL of the response.
+   */
   url(): string {
     return this._url;
   }
 
+  /**
+   * @returns True if the response was successful (status in the range 200-299).
+   */
   ok(): boolean {
+    // TODO: document === 0 case?
```

Comment:
@mathiasbynens do you have any context to this case? I'm not sure why we'd have this.

Reply:
Maybe @sigurdschneider knows?